File: Lógica/Lógic C#/POO/Att11/Models/Gerente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Att11.Models
{
    public class Gerente : Funcionario
    {
        public Gerente (string nome, decimal salario) : base(nome, salario) { }

        public override decimal CalcularSalarioLiquido() {
            return Salario * 1.20m;
        }


    }
}

File: Lógica/Lógic Jav/src/POO/att16/src/Models/Gerente.java
package att16.src.Models;

import att16.src.Enums.Cargos;

public class Gerente extends Funcionario{

    private static final double BONUS = 1.15;

    public Gerente(String nome, double salario) {
        super(nome, salario);
        this.cargo = Cargos.GERENTE;
    }

    @Override
    public double getSalario(){
        return this.salario + (this.salario * BONUS);
    }

    @Override
    public String toString() {
        return "Nome: " + nome +
               " | Cargo: " + cargo +
               " | Salário Base: R$ " + String.format("%.2f", salario) +
               " | Bônus: R$ " + String.format("%.2f", salario * BONUS) +
               " | Salário Final: R$ " + String.format("%.2f", getSalario());
    }

}
